### PR TITLE
chore(apps/interface): Update Footer

### PR DESCRIPTION
**Pull request by math-marcellino**
Update footer styling & test id

### CODE REVIEW

File: apps/interface/components/FooterBar/index.tsx [hunk 1]
```diff
@@ -30,35 +28,34 @@ export const FooterBar = () => {
             <Flex
                 borderTop="1px"
                 borderColor={gray3}
-                py="4"
+                py="5"
+                px="4"
```
Comment (pyk) [inline apps/interface/components/FooterBar/index.tsx:32]: @math-marcellino I think we need to remove the `px` ? coz the footer is not aligned with the rest of content if we add another `px` padding

**Review by pyk — CHANGES_REQUESTED** [review]
Just smol nits

File: apps/interface/components/FooterBar/index.tsx (hunk 1 above)
Comment (math-marcellino) [reply apps/interface/components/FooterBar/index.tsx:32]: Ahh yes you're right. Lemme remove it sir 👍

**Review by pyk — APPROVED** [review]
LGTM @math-marcellino

**Pull request merged**